src/Mod/TechDraw/App/DrawViewPartPy.xml

```diff
@@ -138,16 +138,26 @@
         <UserDocu>formatGeometricEdge(index, style, weight, color, visible). Returns None.</UserDocu>
       </Documentation>
     </Methode>
-        <Methode Name="getEdgeByIndex">
+    <Methode Name="getEdgeByIndex">

```

Comment:
@vocx-fc haven't you said that FreeCAD is moving to more pythonic style, i.e. snake_case? Is there a disagreement between you and @WandererFan ?

Or it's just that FreeCAD's python binding generator doesn't support aliases?

Reply:
Well, the C++ guys do whatever they want. I want to have `snake_case`, I can't control what others do.

Reply:
@vocx-fc But it isn't C++, it's python API.